.travis.yml

```diff
@@ -0,0 +1,21 @@
+script:
+  - pushd source
+  - make -j2
```

Comment:
@tkoeppe Only because I know `make` works and have never tried building it with `latexmk`. If `latexmk` works and is preferred, I'm happy to use it instead.